### PR TITLE
HOUSNAV-89: Modal UI Polish

### PR DESCRIPTION
**Pull request by nicholaslyon-dt**
[HOUSNAV-89](https://hous-bssb.atlassian.net/browse/HOUSNAV-89)

## Overview & Purpose
Update the side modal UI to match the design at all screen sizes

## Summary of Changes
Refactored styles and classes in the side modal for the building code and the glossary

## Side Effects
Possible issues with printing to PDF. I think I checked them all but might need another pair of eyes.

## Testing
Navigate to and use the Building Code and Glossary side modal. Check the PDF print option in the side modal and the results screen.

## Checklist
- [x] I have written or updated vitests for this work
- [x] I have reviewed the [accessibility guidelines](https://digital.gov.bc.ca/wcag/home/) relevant to this work
- [x] I have reviewed this work with at least one screen reader (when applicable)
- [x] I have added/updated any related documentation

### CODE REVIEW

**Comment by wardm89** [comment]
Can you add a print css update so that the building code and glossary terms takes up the whole width of the page when printing to pdf for all cases.

Screenshot 2024-07-24 at 9 40 46 AM
Screenshot 2024-07-24 at 9 42 41 AM

File: packages/ui/src/modal-side/ModalSide.css [hunk 1]
```diff
@@ -39,103 +38,165 @@
...
+.ui-ModalSide--Clauses li:before {
+  right: calc(
+    calc(100% - var(--modal-bc-sentence-padding-start)) +
+      var(--layout-margin-medium)
+  );
+}
+
+.ui-ModalSide--Clauses > li:before {
+  content: counter(countclause, lower-alpha) ".";
+}
```
Comment (wardm89) [inline packages/ui/src/modal-side/ModalSide.css:176]: I learned something cool today :) 

Had no clue you could do this in css!

Comment (nicholaslyon-dt) [reply packages/ui/src/modal-side/ModalSide.css:176]: Yeah I had to pull some tricks out for this stuff 😂!

**Comment by wardm89** [comment]
The tables 
Screenshot 2024-07-24 at 10 28 32 AM

**Review by wardm89 — APPROVED** [review]

**Pull request merged**